### PR TITLE
Updated spanish translation

### PR DESCRIPTION
**Pull request by KewaiiGamer**

### CODE REVIEW

File: src/main/resources/assets/armorplus/lang/es_ES.lang [hunk 1]
```diff
@@ -87,4 +127,11 @@ achievement.armorplus.craftEnderDragonArmor.desc=Mortar el Dragón del End, y cr
 achievement.armorplus.craftGuardianArmor=¡Lleno de Espinos!
 achievement.armorplus.craftGuardianArmor.desc=Crear una Pieza de Armadura de Guardián
 achievement.armorplus.craftTheUltimateArmor=¡El Poder Supremo!
-achievement.armorplus.craftTheUltimateArmor.desc=Crear una Pieza de Armadura Suprema
\ No newline at end of file
+achievement.armorplus.craftTheUltimateArmor.desc=Crear una Pieza de Armadura Suprema
+
+#CONSOLE LOG -- As part of console output, these entries shall not be translated. --
```
Comment (sokratis12GR) [inline src/main/resources/assets/armorplus/lang/es_ES.lang:132]: @KewaiiGamer this part is no longer used and would be best if it is removed

File: src/main/resources/assets/armorplus/lang/es_ES.lang [hunk 2]
```diff
@@ -67,6 +67,46 @@ item.TheUltimateChestplate.name=El Peto Supremo
...
+item.armorplus.lava_battle_axe.name=Hacha de Batalla de Lava
+item.armorplus.super_star_battle_axe.name=Hacha de Batalla de Estrella Estupenda
+item.armorplus.guardian_battle_axe.name=Hacha de Batalla del Guardián
+item.armorplus.ender_dragon_battle_axe.name=Hacha de Batalla del Dragon del End
+
+#Bows
+item.armorplus.coal_bow.name=Arco de Carbón
+item.armorplus.lapis_bow.name=Arco de Lapislázuli
+item.armorplus.redstone_bow.name=Arco de Redstone
+item.armorplus.emerald_bow.name=Arco de Esmeralda
```
Comment (elifoster) [inline src/main/resources/assets/armorplus/lang/es_ES.lang:96]: Shouldn't this be `Arco Esmeralda` or the other emerald things be `Cosa de Esmeralda`?

File: src/main/resources/assets/armorplus/lang/es_ES.lang [hunk 3]
```diff
@@ -67,6 +67,46 @@ item.TheUltimateChestplate.name=El Peto Supremo
...
+item.armorplus.super_star_bow.name=Arco de Estrella Estupenda
+item.armorplus.guardian_bow.name=Arco del Guardián
+item.armorplus.ender_dragon_bow.name=Arco del Dragon del End
+
+#Arrows
+item.armorplus.coal_arrow.name=Flecha de carbón
+item.armorplus.lapis_arrow.name=Flecha de Lapislázuli
+item.armorplus.redstone_arrow.name=Flecha de Redstone
+item.armorplus.lava_arrow.name=Flecha de Lava
+item.armorplus.ender_dragon_arrow.name=Flecha del Dragon del End
```
Comment (elifoster) [inline src/main/resources/assets/armorplus/lang/es_ES.lang:108]: `Dragon` -> `Dragón`

File: src/main/resources/assets/armorplus/lang/es_ES.lang [hunk 4]
```diff
@@ -67,6 +67,46 @@ item.TheUltimateChestplate.name=El Peto Supremo
...
+#Bows
+item.armorplus.coal_bow.name=Arco de Carbón
+item.armorplus.lapis_bow.name=Arco de Lapislázuli
+item.armorplus.redstone_bow.name=Arco de Redstone
+item.armorplus.emerald_bow.name=Arco de Esmeralda
+item.armorplus.obsidian_bow.name=Arco de Obsidiana
+item.armorplus.lava_bow.name=Arco de Lava
+item.armorplus.super_star_bow.name=Arco de Estrella Estupenda
+item.armorplus.guardian_bow.name=Arco del Guardián
+item.armorplus.ender_dragon_bow.name=Arco del Dragon del End
```
Comment (elifoster) [inline src/main/resources/assets/armorplus/lang/es_ES.lang:101]: `Dragon` -> `Dragón`

File: src/main/resources/assets/armorplus/lang/es_ES.lang [hunk 5]
```diff
@@ -67,6 +67,46 @@ item.TheUltimateChestplate.name=El Peto Supremo
...
+#BattleAxes
+item.armorplus.coal_battle_axe.name=Hacha de Batalla de Carbón
+item.armorplus.lapis_battle_axe.name=Hacha de Batalla de Lapislázuli
+item.armorplus.redstone_battle_axe.name=Hacha de Batalla de Redstone
+item.armorplus.emerald_battle_axe.name=Hacha de Batalla Esmeralda
+item.armorplus.obsidian_battle_axe.name=Hacha de Batalla de Obsidiana
+item.armorplus.lava_battle_axe.name=Hacha de Batalla de Lava
+item.armorplus.super_star_battle_axe.name=Hacha de Batalla de Estrella Estupenda
+item.armorplus.guardian_battle_axe.name=Hacha de Batalla del Guardián
+item.armorplus.ender_dragon_battle_axe.name=Hacha de Batalla del Dragon del End
```
Comment (elifoster) [inline src/main/resources/assets/armorplus/lang/es_ES.lang:90]: `Dragon` -> `Dragón`

File: src/main/resources/assets/armorplus/lang/es_ES.lang [hunk 6]
```diff
@@ -67,6 +67,46 @@ item.TheUltimateChestplate.name=El Peto Supremo
...
+#Swords
+item.armorplus.coal_sword.name=Espada de Carbón
+item.armorplus.lapis_sword.name=Espada de Lapislázuli
+item.armorplus.redstone_sword.name=Espada de Redstone
+item.armorplus.emerald_sword.name=Espada Esmeralda
+item.armorplus.obsidian_sword.name=Espada de Obsidiana
+item.armorplus.lava_sword.name=Espada de Lava
+item.armorplus.super_star_sword.name=Espada de Estrella Estupenda
+item.armorplus.guardian_sword.name=Espada de Guardián
+item.armorplus.ender_dragon_sword.name=Espada del Dragon del End
```
Comment (elifoster) [inline src/main/resources/assets/armorplus/lang/es_ES.lang:79]: `Dragon` -> `Dragón`

**Review by elifoster — CHANGES_REQUESTED** [review]

**Comment by sokratis12GR** [comment]
Okay, so soon (within a few days/hours) I'm going to merge this and modify the files with the given suggestions and changes.

**Pull request merged**